packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,26 +7,35 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [74.1.0]
+
 ### Changed
 
 - Bump `@metamask/base-controller` from `^8.1.0` to `^8.2.0` ([#6355](https://github.com/MetaMask/core/pull/6355))
-- Uses the correct internal account type for the asset ([#6358](https://github.com/MetaMask/core/pull/6358)).
+
 - Enable `AccountTrackerController` to fetch native balances using AccountsAPI when `allowExternalServices` is enabled ([#6369](https://github.com/MetaMask/core/pull/6369))
```

Comment:
Should this go in "Added"? This seems like a new feature.

Reply:
`allowExternalServices` is also an option that wasn't there before, and that isn't clear from this entry.